app/src/main/java/a/a/a/Jx.java

```diff
@@ -299,20 +315,21 @@ public String a() {
             }
             sb.append(a);
         }
-        if (!this.f.a(this.c.getActivityName()).a() || this.c.fileName.contains("_fragment")) {
-            sb.append("initializeLogic();");
-            sb.append(a);
-        } else {
+        sb.append(a);
+        if (!this.c.fileName.contains("_fragment")) {
             jq jqVar = this.f;
-            sb.append(Lx.a(jqVar.g, jqVar.a(this.c.getActivityName()).c));
+            //sb.append(Lx.a(jqVar.g, jqVar.a(this.c.getActivityName()).c));
```

Comment:
Just a hint: you don't have to comment out old code, git will automatically save what was on that line before you made a commit.

Reply:
I commented out when I am testing and I forgot about that